src/libraries/System.Net.Http/src/System/Net/Http/BrowserHttpHandler/BrowserHttpHandler.cs

```diff
@@ -0,0 +1,590 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Should that be in a separate folder? 

@stephentoub thoughts?

Reply:
I think it's reasonable, especially if a) we expect there to be multiple files per handler, and b) we expect there to be multiple handlers added (which I believe we do).